iml-system-test-utils/src/vagrant.rs

```diff
@@ -58,12 +58,25 @@ pub async fn up<'a>() -> Result<Command, CmdError> {
     Ok(x)
 }
 
-pub async fn destroy<'a>() -> Result<(), CmdError> {
-    let mut x = vagrant().await?;
+pub async fn destroy<'a>(config: &ClusterConfig) -> Result<(), CmdError> {
+    let mut nodes = config.all();
+    nodes.reverse();
```

Comment:
This seems brittle, we are making the assumption that reversing the config will always present nodes in an order suitable for shutdown.

Perhaps it would be better to make this a method on the `ClusterConfig`. That way, the intent is clear and it has a better chance being noticed in future updates.

Reply:
sure 👍